client/js/app/main/modules/week/week.controller.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @ngInject
+ */
+module.exports = function (todayService, $filter, nowValue) {
+    var vm = this,
+        oneDay = 86400000,
+        parsedDate = Date.parse(nowValue);
+    this.tasks = [];
+    this.todayLong = $filter('date')(nowValue, "d MMM yyyy").toLowerCase();
+
+    this.weekDays = [
```

Comment:
Move logic of date manipulation from controller to service, if we have no service for it - create it.